### ISSUE TITLE
"Permission denied (publickey)" when running "flintrock login test-cluster"

### ISSUE THREAD

**Issue by dado0583** [nchammas/flintrock#210]
If you are reporting a potential bug, include as many of the following
as are relevant (and delete this sentence, of course):
* Flintrock version: 0.9.0
* Python version: 3.6
* OS: OSX

I have a fairly basic issue I think. I created a cluster. I can see the ec2 instances listed on the AWS after launching. When I try to login I get an SSH type error. The IP listed below is the master IP address. Is there something I am missing from the config?

```
$ flintrock login test-cluster
Warning: Permanently added '54.209.216.179' (ECDSA) to the list of known hosts.
Permission denied (publickey).
```

I'm confused. I can tell the .pem is getting picked up because the cluster is being created. When I try to hit the master node <master-ip>:8080 I can't connect. Is there an additional networking step I need to take?

**Comment by nchammas** [comment]
Have you successfully used the same `.pem` file to login to an EC2 instance before? If not, could you test that out? It may be an issue with the file, like incorrect permissions.

**Comment by nchammas** [comment]
Closing this due to inactivity. If you have an update @dado0583 feel free to reopen!
